k8s/gx-portal-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
```

Comment:
can you move this to gx-deployment project?

Reply:
To be honest, I would agree to have it in a different project if we didn't have to deploy it manually. It makes things harder to have to search in a different project files belonging to this one.

Reply:
They have to be in a different project. This is code that will need to be merged upstream. Infra specific descriptors should never live directly with source code. Bad idea all around

Reply:
I made comments about not using this repo for our infra before...

Reply:
Fixed!